temporal-sdk/build.gradle

```diff
@@ -34,3 +34,15 @@ task registerNamespace(type: JavaExec) {
 }
 
 test.dependsOn 'registerNamespace'
+
+test {
+    useJUnit {
+        excludeCategories 'io.temporal.worker.IndependentResourceBasedTests'
+    }
+}
+
+task testResourceIndependent(type: Test) {
+    useJUnit {
+        includeCategories 'io.temporal.worker.IndependentResourceBasedTests'
```

Comment:
I wonder if you want  to disable concurrent tests as well  here in case you add more tests in the future

Reply:
Ya good call